src/Database/LSMTree/Internal/BlobFile.hs

```diff
@@ -0,0 +1,105 @@
+{-# LANGUAGE DeriveAnyClass     #-}
+{-# LANGUAGE DeriveGeneric      #-}
+{-# LANGUAGE DerivingStrategies #-}
```

Comment:
`DeriveGeneric` seems unused, and the others are enabled by default, I think

Reply:
Copy'n'paste :smile:

Reply:
Removed them all.